### ISSUE TITLE
Accessibility issues with Font Awesome

### ISSUE THREAD

**Issue by roelvangils** [digipolisantwerp/acpaas-ui_angular#227]
The Modal, Navigation and Datepicker component all use icons.

### Failures:

- The icons don't have a text alternative. This results in things like unnamed interactive elements (WCAG 4.1.2). See [alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) for more info.
- Images are not marked up as images. Refer to [failure F87](https://www.w3.org/WAI/WCAG21/Techniques/failures/F87.html) for the failure and [ARIA Technique 24](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA24.html) for a possible solution. Advisory: using self-hosted images is even more robust and lighter for the user.

This doesn't satisfy [1.1.1 Non-text Content](https://wcag.be/111#non-text-content), [1.3.1 Info and Relationships](https://wcag.be/131#info-and-relationships) and [4.1.2 Name, Role, Value](https://wcag.be/412#name-role-value).

Note: this issue is (potentially blocking) and applies to all components that use Font Awesome icons.
